azure-javascript/package.json

```diff
@@ -3,6 +3,6 @@
     "main": "index.js",
     "dependencies": {
         "@pulumi/pulumi": "^3.0.0",
-        "@pulumi/azure-native": "^1.0.0"
+        "@pulumi/azure-native": "v2.0.0"
```

Comment:
Why have we decided to pin `v2.0.0` instead of any value in `v2.*` (which is what `^1.0.0` did before)?

This is the same question as for the .NET version.